### ISSUE TITLE
NamedTuple ._make does not detect type violations

### ISSUE THREAD

**Issue by pkch** [python/mypy#3123]
(based on #3111)

This passes type check:

```
X = NamedTuple('X', [('x', int), ('y', str)])
X._make('a b')
```

Simple type check passes because string is an `Iterable` (the runtime only detects a problem if the string length is not the same as the number of namedtuple fields).

We could, however, check types of items in the `Iterable` in case it's a literal; alternatively, we can simply remove the corresponding test (`testNamedTupleMake`).

**Comment by JukkaL** [comment]
Maybe we should just update the test to work consistently. For example, pass it an integer instead of a string.

**Comment by JukkaL** [comment]
There is still the issue of `_make` having an imprecise signature, which is a problem though not a high-priority one since there is no easy way to represent the precise signature right now.

**Comment by hauntsaninja** [comment]
Closing this as won't fix, doesn't seem to have been much demand for it
